src/main/java/core/basesyntax/impl/StorageImpl.java

```diff
@@ -1,20 +1,48 @@
 package core.basesyntax.impl;
 
 import core.basesyntax.Storage;
+import java.util.Objects;
 
 public class StorageImpl<K, V> implements Storage<K, V> {
+    private static final int MAX_SIZE = 10;
+    private int sizeOfStorage = 0;
```

Comment:
```suggestion
    private int sizeOfStorage;
```
по дефолту інтова змінна рівна 0, тому можемо прибрати ініціалізацію

Reply:
add